addons/point_of_sale/i18n/point_of_sale.pot

```diff
@@ -5648,6 +5648,14 @@ msgstr ""
 msgid "You cannot close the POS when orders are still in draft"
 msgstr ""
 
+#. module: point_of_sale
+#: code:addons/point_of_sale/models/pos_session.py:0
+#, python-format
+msgid ""
```

Comment:
this is a manuall modification of POT file, it should be exported from the database, to be correctly integrated and avoid noise in future export and ease the forward port

Reply:
Done